deploy/examples/csi/cephfs/storageclass.yaml

```diff
@@ -14,7 +14,7 @@ parameters:
 
   # Ceph pool into which the volume shall be created
   # Required for provisionVolume: "true"
-  pool: myfs-data0
+  pool: myfs-replicated
```

Comment:
We should change `myfs-data0` in other manifests (like `nfs.yaml` and `nfs-test.yaml`) also. Otherwise, the NFS examples won't successfully run with the new filesystem manifests.

Reply:
Ok, updated.